packages/core/src/PlayerContextProvider.js

```diff
@@ -631,13 +631,14 @@ export class PlayerContextProvider extends Component {
     }
   }
 
-  seekComplete() {
+  seekComplete(targetTime) {
     const { seekPreviewTime, awaitingResumeOnSeekComplete } = this.state;
     const baseStateUpdate = {
       seekInProgress: false,
       awaitingResumeOnSeekComplete: false
     };
-    if (isNaN(seekPreviewTime)) {
+    const currenTime = targetTime ? targetTime : seekPreviewTime
```

Comment:
Since `seekComplete` could be passed somewhere as an event handler I think we need to make sure `typeof targetTime === 'number'` rather than just checking if `targetTime` is truthy (sometimes we could be passed an argument, but it's an event, not a number).

Reply:
Looks like we've got a typo here; it should be `currentTime` not `currenTime` 🙂. Also I think we need a semicolon at the end to make prettier happy. 😄

Reply:
Should be good now!